src/main/webapp/app/exercises/shared/plagiarism/types/modeling/ModelElement.ts

```diff
@@ -0,0 +1,3 @@
+import { PlagiarismSubmissionElement } from '../PlagiarismSubmissionElement';
+
+export class ModelElement extends PlagiarismSubmissionElement {}
```

Comment:
```suggestion
export class ModelPlagiarismElement extends PlagiarismSubmissionElement {}
```

Reply:
can you add some attributes here as well? I guess we need at least a referenceId

Reply:
If we assume the elements' order does not change, we can use the index to reference specific elements of a submission (compare with attributes of PlagiarismMatch). Since modeling elements don't have a strictly sequential order, the `length` of each match could be 1 everytime.